packages/admin/cms-admin/src/redirects/RedirectActiveness.tsx

```diff
@@ -59,6 +59,7 @@ const RedirectActiveness = ({ redirect }: RedirectActivenessProps): JSX.Element
                     active: active,
                 },
             },
+            refetchQueries: ["PaginatedRedirects"],
```

Comment:
instead of the refetch you could query the changed field (activatedAt) or even use optimisticResponse for it